### ISSUE TITLE
Property 'children' does not exist on type

### ISSUE THREAD

**Issue by stereodenis** [nysamnang/react-native-raw-bottom-sheet#163]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-raw-bottom-sheet@2.2.0` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-raw-bottom-sheet/index.d.ts b/node_modules/react-native-raw-bottom-sheet/index.d.ts
index 31a5259..1c4cf13 100644
--- a/node_modules/react-native-raw-bottom-sheet/index.d.ts
+++ b/node_modules/react-native-raw-bottom-sheet/index.d.ts
@@ -1,4 +1,4 @@
-import { Component } from "react";
+import { Component, PropsWithChildren } from "react";
 import { StyleProp, ViewStyle } from "react-native";
 
 declare module "react-native-raw-bottom-sheet" {
@@ -20,7 +20,7 @@ declare module "react-native-raw-bottom-sheet" {
       draggableIcon?: StyleProp<ViewStyle>;
     };
     keyboardAvoidingViewEnabled?: boolean;
-  };
+  } & PropsWithChildren;
 
   export default class RBSheet extends Component<RBSheetProps> {
     open(): void;
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by msachi** [comment]
@nysamnang @stereodenis Worth making a PR to fix this for everyone?

**Comment by mccraveiro** [comment]
@msachi unfortunately this projects seems to be abandoned

**Comment by nysamnang** [comment]
Merged #146
